examples/complete/main.tf

```diff
@@ -1,7 +1,24 @@
-module "example" {
+module "label" {
+  source     = "cloudposse/label/null"
+  version    = "0.25.0"
+  attributes = [var.function_name]
+
+  context = module.this.context
+}
+
+data "archive_file" "lambda_zip" {
+  type        = "zip"
+  source_file = "handler.js"
+  output_path = "lambda_function.zip"
+}
```

Comment:
Can we delegate this to the root module?

See: https://github.com/cloudposse/terraform-aws-cloudfront-s3-cdn/tree/master/modules/lambda%40edge#lambdaedge

Reply:
I don't think we want to be prescriptive on how to create the zip file in this module. For instance, it could contain multiple files, dependencies (such as `node_modules`), etc. We should leave it up to the caller to determine _how_ the zip file gets built.